CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## 2.1.0 (TBD)
+
+### Breaking Changes
```

Comment:
Let's call this a deprecation, as we're not classing this as breaking (if we were, it would be v3.0.0).

```suggestion
### Deprecations
```